src/templates/macros/task_macros.html.twig

```diff
@@ -0,0 +1,47 @@
+{% macro taskListingCurrent(tasksCurrent) %}
+<section class="listing current_week content-flex-item">
```

Comment:
It would be better to name classes consistently (so not `some_class` and `some-class` at the same time)